cmd/world/root/root.go

```diff
@@ -44,6 +52,63 @@ func init() {
 	logger.AddLogFlag(doctorCmd)
 }
 
+func checkLatestVersion() error {
+	url := "https://api.github.com/repos/Argus-Labs/world-cli/releases/latest"
+
+	// Create a request with the context
+	req, err := http.NewRequestWithContext(context.Background(), http.MethodGet, url, nil)
+	if err != nil {
+		return fmt.Errorf("error creating request: %s", err.Error())
+	}
+
+	// Create a new HTTP client and execute the request
+	client := &http.Client{}
+	resp, err := client.Do(req)
```

Comment:
- Add timeout for the HTTP request so the user isn't stuck waiting if there's no network connection
- If it fails, just stay silent. We don't want want this to be disruptive if GitHub API is having an issue or the user doesn't have internet. If you want, you can log.Debug() them, but don't print/log errors related to the version check by default.

Reply:
added 2 seconds time out and return nil if there's an error